### PR TITLE
Database Service + LLAMA Service

### PR DESCRIPTION
**Pull request by garipovroma**
Database Service + LLAMA Service implemented, also tests for database service are provided

### CODE REVIEW

File: tests/test_llama_service.py
Comment (ST-1580) [inline tests/test_llama_service.py:1]: I think file can be removed

File: tests/test_database.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import sys
+import os
+
+from services.database_service import DatabaseService
+import sqlite3
+import pytest
+
```
Comment (ST-1580) [inline tests/test_database.py:7]: What about `sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))`?

File: scripts/fill_db.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import sys
+import os
+
+from services.database_service import DatabaseService
+
```
Comment (ST-1580) [inline scripts/fill_db.py:5]: What about `sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))`?

File: examples/llama_service_example.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import sys
+import os
+
+from services import LlamaService
+
```
Comment (ST-1580) [inline examples/llama_service_example.py:5]: What about `sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))`?

File: tests/test_llama_service.py
Comment (garipovroma) [reply tests/test_llama_service.py:1]: Done

**Pull request merged**